Projects/Metatron_Protocol_Launch/README.md

```diff
@@ -0,0 +1,69 @@
+# Project Metatron
+
+Welcome to Project Metatron.
+
+## Project Goal
+
+The Goal of Project Metatron is to successfully launch the [Metatron Protocol](https://github.com/macterra/metatron/blob/main/docs/README.md) to the public as per the [Open Grant Proposal](https://github.com/Flaxscrip/devgrants/blob/Flaxscrip-patch-1/open-grant-proposals/open-proposal-metatron.md) submitted to the [Filecoin Foundation](https://github.com/filecoin-project/devgrants) in August 2021.
+
+## Ethical Data Pilot
+
+The Metatron Protocol provides key functionality that will be immediately put to use in a technology pilot involving members from the [Ethical Data Alliance](https://ethicaldata.net/about-us/governance/) community.
+
+A full description of the EDA's needs for peer-to-peer technology is available online: [Ethical Data Alliance - What we are doing](https://ethicaldata.net/what-we-are-doing/).
+
+## EDEN Use Cases
+
+In summary, the EDA community needs Metatron and Filecoin to build a data exchange network that protects data authorship and history. We call these use cases the ["EDEN Protocol"](https://edenprotocol.io).
+
+The 4 main requirements of the EDEN Protocol are:
+
+1. Data Notary: Document proof-of-existence (IPFS cid & Metatron xid)
+1. Data Catalog: Document storage (IPFS & Filecoin)
+1. Data Reputation: Document history traceability (Metatron xid)
+1. Data Exchange: Document versioning rights transfer (Metatron transfer)
+
+The EDEN Protocol will become a set of instructions, customizations, and lessons learned on how to deploy the Metatron & Filecoin environments in order to participate in the data exchange community being defined by the EDA.
+
+## Web3 Supply Chain Management
+
+At the outset of the Metatron Launch and EDEN Pilot, we will have implemented and demonstrated a P2P protocol for supply chain management and product traceability. While our initial user community is built around the EDA, this Web3 Supply Chain Management and Product Traceability solution will be ready to engage with other industries (food, art, medicine, etc).
```

Comment:
Maybe:
> The goals of the Metatron Launch and EDEN Pilot are to implement and demonstrate a P2P protocol for supply chain management and product traceability.

Reply:
Great suggestion. Integrated in next commit.